### PR TITLE
Muffasa story

### PR DESCRIPTION
**Pull request by Muffasa**
let me know if i miss understood and i should have done everyting on the master, and fell free to edit the text if somthing is wrong in any range. amaizing course btw!

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/lucidream/luciddream.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+As you see the door opens, while you well aware that the spell you just cast is
```
Comment (casey-collab) [inline english/lucidream/luciddream.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Muffasa** [comment]
Thanks for leting me know there was no option to get to my addition,
amaizing course btw! keep up the great work=]

On Thu, Jun 4, 2015 at 9:11 PM, Casey Collaborator <notifications@github.com
> wrote:

> Besides my other comments, this looks good! Once you respond to those,
> this should be good to merge.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/udacity/create-your-own-adventure/pull/1492#issuecomment-108996042>
> .
>



-- 
בברכה,

אהד גלאור
0544637999
ohadgalor1990@gmail.com
ohadg@n-i-p.co.il

**Comment by cbuckey-uda** [comment]
You're welcome! I'm glad you enjoyed the course. @casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :smile:

**Pull request merged**